support-frontend/app/controllers/DigitalSubscriptionController.scala

```diff
@@ -72,10 +72,16 @@ class DigitalSubscriptionController(
         val maybePromotionCopy = {
           landingCopyProvider.promotionCopy(promoCodes ++ defaultPromos, DigitalPack, "uk", orderIsAGift)
         }
+
+        val mainElement = assets.getSsrCacheContentsAsHtml(
+          divId = s"digital-subscription-checkout-page-$countryCode",
+          file = "digital-subscription-checkout-page.html",
+        )
+
         Ok(
           views.html.subscriptionCheckout(
             title,
-            id,
```

Comment:
Can we delete the `val` now too? I can't see where it's used.

Reply:
yes, cleaned up